csrc/ops.cu

```diff
@@ -693,9 +673,9 @@ template <typename T> void gemm_host(int m, int n, int k, T * A,  T* B,  T * out
 	//cout << m << endl;
 	//cout << n << endl;
 	//cout << k << endl;
-  //if(bits == 32)
+  if(bits == 32)
     //gemm_device<T, 32, 128><<< num_blocks, 128, 0, 0 >>>(m,  n,  k, A,  B,  out, lda, ldb, ldc);
-    //gemm_device<T, 32, 32><<< num_blocks, 32, 0, 0 >>>(m,  n,  k, A,  B,  out, lda, ldb, ldc);
+    gemm_device<T, 32, 32><<< num_blocks, 32, 0, 0 >>>(m,  n,  k, A,  B,  out, lda, ldb, ldc);
```

Comment:
Is there a particular issue this is addressing? I think we're likely to deprecate this anyway, but would be good to know more about expected impact in the meantime.

Reply:
Yes I was thinking about removing the 32 bit gemm_device template altogether as well. Similar case is in gemm_4 bit_inference . Since the template implementation is there in kernels.cu, I had uncommented this . But in reality this & the equivalent template invocation can be done away with . What do you suggest?

Reply:
I'm not opposed to keeping it for now and taking another look in a follow up PR that I'm planning. In that PR `gemm_4bit_inference` is going to be removed.

Reply:
> I'm not opposed to keeping it for now and taking another look in a follow up PR that I'm planning. In that PR `gemm_4bit_inference` is going to be removed.

In that case, I would suggest to keep it for now. BTW, will the interface of gemm_4bit_inference be changed? 
I think this PR is complete, barring just the nit format which I am updating.
Also a note that I might be picking up the cutlass port soon for the gemms, just going through some housekeeping for now.

Reply:
The lint should be fixed now . @matthewdouglas